### ISSUE TITLE
Feature request: Add track search

### ISSUE THREAD

**Issue by adamijak** [xou816/spot#414]
Expected behaviour: 
When searching song results should be shown next to albums and artists category. 
After playing selected song Spot should continue playing. Probably song radio?

**Comment by Staini** [comment]
As workaround I start the spotify android app, to search for a special song.
Including songs to the search function would be great.

**Comment by xou816** [comment]
Yes, I need to redo search from scratch, many things are missing here! :)

**Comment by AbrarSL** [comment]
For a lot of the issues based around missing features it might be a good idea to take a look at [Psst](https://github.com/jpochyla/psst). It's also written in Rust and makes use of Librespot. It has a lot of the features that people are requesting but it is of course not as good looking as Spot ;D
